docs/sources/write/style-guide/word-list/index.md

```diff
@@ -74,8 +74,13 @@ Use this rather than _hold the pointer over_ or _point to_.
 
 Use this rather than _hamburger menu_ or _kebab menu_.
 
-<!--
 ## N
+
+### Node Explorer
+
+When referring to the Prometheus Node Explorer, capitalize both words in the term _Node Explorer_. Don't use _Node explorer_ or _node explorer_.
```

Comment:
```suggestion
### Node Exporter

When referring to the Prometheus Node Exporter, capitalize both words in the term _Node Exporter. Don't use _Node exporter or _node exporter.
```

Reply:
LOL! Yes. This is my brain on dumb mode.